gradle.properties

```diff
@@ -19,8 +19,6 @@ gradle.internal.testdistribution.writeTraceFile=true
 develocity.internal.testdistribution.writeTraceFile=true
 gradle.internal.testdistribution.queryResponseTimeout=PT20S
 develocity.internal.testdistribution.queryResponseTimeout=PT20S
-# Disable GE -> DV deprecation warnings, see https://github.com/gradle/gradle-org-conventions-plugin/issues/26
-systemProp.develocity.deprecation.muteWarnings=true
```

Comment:
🤔 Hmm... There still seem to be some warnings on CI: https://ge.gradle.org/s/ubhejd4wegkps/console-log?page=3#L2025-L2027
Did we update the TC scan plugin already?